packages/medusa/src/helpers/test-request.js

```diff
@@ -87,16 +88,18 @@ export async function request(method, url, opts = {}) {
   )
   headers.Cookie = headers.Cookie || ""
   if (opts.adminSession) {
-    if (opts.adminSession.jwt) {
-      opts.adminSession.jwt = jwt.sign(
-        opts.adminSession.jwt,
+    const adminSession = clone(opts.adminSession)
```

Comment:
🙏 indeed, the reference was mutated.
Todo: instead of using the clone util from lodash, I suggest using the spread operator as we can avoid using lodash for that. 
I think that `structuredClone` would work in that case since we only have some string properties and no functions or prototype chain to take into account etc. But we usually work with the spread operator